src/components/Header/utils/index.ts

```diff
@@ -0,0 +1,11 @@
+import { TITLE } from '../constants';
+
+export const getPathname = (segment: number) => {
```

Comment:
```suggestion
export const getNthPathname = (segment: number) => {
```
변수명을 이렇게 해주시면 가독성이 조오오금 더 좋지 않을까? 라는 생각이 들었습니다 !